### PR TITLE
New package: ahrm.sioyek version 2.0.0

### LINKED ISSUE #140029: [Package Request]: sioyek
### How can we help?

I would like someone else to build the manifest.

### Please read and ensure the following

- [X] The installer meets the above requirements

### Please provide the following information

Download Page Url: https://github.com/ahrm/sioyek/releases/latest
Publisher: https://github.com/ahrm
Package Name: sioyek
Description: Sioyek is a PDF viewer with a focus on technical books and research papers
Package Version: 2.0.0
Installer URL: https://github.com/ahrm/sioyek/releases/tag/v2.0.0

### PR DESCRIPTION
**Pull request by Exorcism0666**
- [X] Resolve: #140029

---

### CODE REVIEW

**Comment by stephengillie** [comment]
Manual Validation ended with:
image

File: manifests/a/ahrm/sioyek/2.0.0/ahrm.sioyek.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: ahrm.sioyek
+PackageVersion: 2.0.0
+InstallerType: zip
+NestedInstallerFiles:
+- RelativeFilePath: sioyek-release-windows\sioyek.exe
```
Comment (stephengillie) [inline manifests/a/ahrm/sioyek/2.0.0/ahrm.sioyek.installer.yaml:8]:
```suggestion
- RelativeFilePath: sioyek-release-windows\sioyek.exe
Dependencies:
  PackageDependencies:
     - PackageIdentifier: Microsoft.VCRedist.2015+.x64
```

**Comment by stephengillie** [comment]
With the dependency, the tutorial PDF doesn't display:

image

**Comment by Exorcism0666** [comment]
> With the dependency, the tutorial PDF doesn't display:
> 
> image

@smopucilowski

**Comment by smopucilowski** [comment]
@ahrm thoughts?

**Comment by ahrm** [comment]
Hmmm maybe it is an OpenGL error? We do need at least opengl 3.3 .

**Comment by stephengillie** [comment]
- https://github.com/microsoft/winget-pkgs/issues/107639
- https://github.com/microsoft/winget-pkgs/issues/12906

**Review by stephengillie — APPROVED** [review]

**Pull request merged**